comfy_panel/score.lua

```diff
@@ -420,6 +478,31 @@ local function on_entity_died(event)
                 color = player.chat_color,
             })
         end
+        
+        if firstplayer and do_loot then
+            firstplayer = false
+            
+            local lootname = LootRaffle.roll_loot(entity.quality.name)
+            local loot_quality = 'uncommon'
+            if lootname == 'railgun' then
+                loot_quality = 'normal'
+            else
+                if entity.quality.name == 'legendary' then
+                    if math_random(1,5) == 1 then
+                        if math_random(1,5) == 1 then
+                            loot_quality = 'legendary'
+                        else
+                            loot_quality = 'epic'
+                        end
+                        player.print('Received ' .. loot_quality .. ' [img=item/' .. lootname .. ']' .. lootname:gsub('-', ' ') .. ' for killing ' .. entity.quality.name .. ' [img=entity/' .. entity.name .. ']' .. entity.name:gsub('-', ' ') .. ' at ' .. string.format("[gps=%s,%s,%s]", entity.position.x, entity.position.y, entity.surface.name))
```

Comment:
there is entity.localised_name maybe better than entity.name:gsub() ?

Also can use localized string/print with something like `print {'', 'entity-name.' .. entity.name }` which is supposedly faster than using lua `..` for concatenation.

Reply:
hmm i removed this line and addded a different one elsewhere instead, the lines shouldn't happen more than once per minute really so i dont think this will eat into performance or cause noticable stutter

Reply:
Agree mainly suggesting to use entity.localised_name instead of gsub here.